### PR TITLE
Stricter typescript configuration

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/utils/@oasisprotocol/ledger.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// TODO: remove and add types upstream
+declare module '@oasisprotocol/ledger' {
+  var x: any
+  export = x
+}
```
Comment (lukaw3d) [inline src/utils/@oasisprotocol/ledger.d.ts:5]: TODO

Comment (buberdds) [reply src/utils/@oasisprotocol/ledger.d.ts:5]: Are we waiting for this TODO to be resolved in this PR or in a following one? not sure this PR is ready for review or not 😄

Comment (lukaw3d) [reply src/utils/@oasisprotocol/ledger.d.ts:5]: Oops :) Ready for review

I think adding types to that upstream library will take a bit

File: src/types/env.d.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+declare namespace NodeJS {
+  export interface ProcessEnv {
+    REACT_APP_BACKEND: 'oasismonitor' | 'oasisscan'
+    REACT_APP_BYPASS_LOCAL: '1' | undefined
```
Comment (buberdds) [inline src/types/env.d.ts:4]: should we define other envs here as well ?

```
REACT_APP_BUILD_DATETIME=
REACT_APP_BUILD_VERSION=
REACT_APP_META_CSP=
```

**Review by buberdds — APPROVED** [review]

File: src/types/env.d.ts (hunk 2 above)
Comment (buberdds) [reply src/types/env.d.ts:4]: aa nvm they are dynamic :D

File: src/utils/@oasisprotocol/ledger.d.ts (hunk 1 above)
Comment (lukaw3d) [reply src/utils/@oasisprotocol/ledger.d.ts:5]: Created https://github.com/oasisprotocol/ledger-js/pull/415

**Pull request merged**